sw-precache-config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  runtimeCaching: [
+    {
+      urlPattern: /^https?:\/\/(.*?).cloudfront.net\/bundles\/(.*?).js$/,
+      handler: 'networkFirst'
```

Comment:
Here is the API documentation (scroll down to _Handler_): https://googlechrome.github.io/sw-toolbox/docs/master/tutorial-api.html

In my opinion it does not make a huge difference if `networkFirst` or `cacheFirst` since we won't have the bundled JS file in cache at first anyway.